langchain/chains/conversational_retrieval/base.py

```diff
@@ -116,9 +117,35 @@ class ConversationalRetrievalChain(BaseConversationalRetrievalChain, BaseModel):
     """Chain for chatting with an index."""
 
     retriever: BaseRetriever
+    """Index to connect to."""
+    reduce_k_below_max_tokens: bool = False
+    """Reduce the number of results to return from store based on tokens limit"""
+    max_tokens_limit: int = 3375
```

Comment:
can we just make this `Optional[int] = None` and if its not None then use it? or is there some downside to that im not thinking of?

Reply:
imo the fewer nobs to turn the better. can leave in other place for backwards compatability but this is better goig forward. happy to change as well if you dont have time today

thanks for adding this!

Reply:
happy to make that change